AndroidManifest.xml

```diff
@@ -12,6 +12,8 @@
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE" />
     <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 
+    <uses-permission android:name="com.android.launcher.permission.INSTALL_SHORTCUT" />
```

Comment:
Nice! But see `Added in API level 19`. So you can only use this on this API level and need to check before on what API you are.